airtime_mvc/application/forms/LiveStreamingPreferences.php

```diff
@@ -125,12 +138,12 @@ public function updateVariables()
                 array ('ViewScript',
                     array (
                       'viewScript'                  => 'form/preferences_livestream.phtml',
-                      'master_source_host'          => isset($masterSourceParams["host"])?$masterSourceParams["host"]:"",
-                      'master_source_port'          => isset($masterSourceParams["port"])?$masterSourceParams["port"]:"",
-                      'master_source_mount'         => isset($masterSourceParams["path"])?$masterSourceParams["path"]:"",
-                      'show_source_host'            => isset($showSourceParams["host"])?$showSourceParams["host"]:"",
-                      'show_source_port'            => isset($showSourceParams["port"])?$showSourceParams["port"]:"",
-                      'show_source_mount'           => isset($showSourceParams["path"])?$showSourceParams["path"]:"",
+                      'master_source_host'          => isset($masterSourceHost)?$masterSourceParams["host"]:"",
+                      'master_source_port'          => isset($masterSourcePort)?$masterSourceParams["port"]:"",
+                      'master_source_mount'         => isset($masterSourceMount)?$masterSourceParams["path"]:"",
+                      'show_source_host'            => isset($showSourceHost)?$showSourceParams["host"]:"",
+                      'show_source_port'            => isset($showSourcePort)?$showSourceParams["port"]:"",
+                      'show_source_mount'           => isset($showSourceMount)?$showSourceParams["path"]:"",
                       'isDemo'                      => $isDemo,
                     )
```

Comment:
Yeah. the code does seem to be working. I'm not sure that we need to overhaul it at this point. If someone wants to spend the time picking it apart it could definitely use refactoring as it is kind of cludgy. The only way I could get the log data from airtime-playout though was to run it sudo -u www-data airtimeplayout from the CLI. The logs for pypo don't appear to be working as they only have a few lines of data.